### PR TITLE
Fix SDK style overridden by CSS reset

### PR DESCRIPTION
**Pull request by WiNloSt**
Fix CSS reset override SDK components

This removes `:where(${baseStyle})` because it doesn't work. We're supposed to add `:where()` to selectors inside `baseStyle` not wrapping all of them inside `:where()` like this. You could see that button doesn't have any inheritance chain from `baseStyle`

https://github.com/metabase/metabase/blob/a536c2d7bf24d111cace04abeedff5f33ac751d5/frontend/src/metabase/css/core/base.styled.ts#L75-L80

#### Master
image

image

#### With `:where(${baseStyle})` removed

Searching `:where(` didn't get the same style we got in `master` to confirm it's already removed.

image

The style chain looks the same that means it didn't work.

#### Before (screenshot added when the changed line was just `${baseStyle}` instead of `:where(${baseStyle})`
Screenshot 2024-04-26 at 11 01 15 PM

#### After
Screenshot 2024-04-26 at 10 57 54 PM

### CODE REVIEW

File: enterprise/frontend/src/embedding-sdk/components/private/SdkContentWrapper.tsx [hunk 1]
```diff
@@ -16,11 +16,9 @@ export const SdkContentWrapper = styled.div<{ font: string; baseUrl?: string }>`
   ${aceEditorStyles}
   ${saveDomImageStyles}
   ${({ theme }) => getRootStyle(theme)}
-  ${baseStyle}
```
Comment (WiNloSt) [inline enterprise/frontend/src/embedding-sdk/components/private/SdkContentWrapper.tsx:19]: This only contains a bunch of CSS reset, I think it's not a bad idea to leave them out for now until we can figure a better solution.

**Review by deniskaber — DISMISSED** [review]
I added my proposed fix in other PR - https://github.com/metabase/metabase/pull/41860/files#diff-14641f26ed97bb265f70dbec7e345518ff6a3e02cf1d54a86102f41f3a44a189R19

I think, it would be slightly better than just removing styles. WDYT?

**Comment by WiNloSt** [comment]
> I added my proposed fix in other PR - https://github.com/metabase/metabase/pull/41860/files#diff-14641f26ed97bb265f70dbec7e345518ff6a3e02cf1d54a86102f41f3a44a189R19
> 
> I think, it would be slightly better than just removing styles. WDYT?

@deniskaber That didn't actually work. `baseStyle` isn't something that could be wrapped in `:where()` alone, we need to apply `:where()` to each selector in `baseStyle` instead. I only [mentioned that on Slack](https://metaboat.slack.com/archives/C06FCQT0KMZ/p1714147391788559?thread_ts=1714038540.138239&cid=C06FCQT0KMZ). So I think removing it is more correct.

**Review by heypoom — APPROVED** [review]
As the styles wrapped in `:where()` did not work at all, I think it's good to remove them for now. Further fixes (if needed) can be added in later PRs.

**Pull request merged**